### PR TITLE
Chore / Add pull request template

### PR DESCRIPTION
**Pull request by styrelius**
### What?

Add a template for pull request descriptions.

<br>

### Why?

A more extensive PR description is beneficial in most cases since it makes it easier (and therefore quicker) to review the changes made. A template urges you to actually add that description, as well as make the description more structured without you having to think about how to structure it on every single occasion.

<br>

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+<br>
+
+### Why?
+
+<!-- Why are these changes made? -->
+
+<!-- If there is one, please link to the ticket where this issue is described. -->
+
+_Referenced ticket [here]()_
```
Comment (palmenhq) [inline .github/PULL_REQUEST_TEMPLATE.md:14]: Maybe we should put this in the PR title and or/branch name instead? So the gitub-jira integration can pick it up

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+### What?
+
+<!-- What changes are made? -->
+<!-- (if there are many significant changes, a list might be a good format)  -->
+
+<br>
```
Comment (palmenhq) [inline .github/PULL_REQUEST_TEMPLATE.md:6]: I feel like i'm just gonna erase these tags every time, can we remove them? Or nah?

**Review by palmenhq — DISMISSED** [review]
Awesome! Just some thoughts

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (palmenhq) [reply .github/PULL_REQUEST_TEMPLATE.md:14]: But if so i think we should put a reminder in the comments here!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+### What?
```
Comment (gustaveen) [inline .github/PULL_REQUEST_TEMPLATE.md:1]:
```suggestion
## What?
```
Just personal preference from me, but I like the h2 with a underline 😃

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (gustaveen) [reply .github/PULL_REQUEST_TEMPLATE.md:6]: True, since you will have to edit it anyway maybe remove?

**Comment by gustaveen** [comment]
Nice @styrelius!👌

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (styrelius) [reply .github/PULL_REQUEST_TEMPLATE.md:14]: I think we could have both maybe? It's nice with a link I think, so it's easy to get to the ticket, but I agree that we should add it to the titles!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (styrelius) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: Haha OK, I don't have a strong preference so I can change it!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (styrelius) [reply .github/PULL_REQUEST_TEMPLATE.md:6]: I mean they're not visible so you don't _have_ to remove them, but sure. Would you like to remove all of the comments? I think the last one that tells you to add screenshots/GIF's might serve a purpose, no?

Comment (palmenhq) [reply .github/PULL_REQUEST_TEMPLATE.md:6]: I ment the `<br>` tag - sorry if that was unclear. The comments i think make sense

Comment (styrelius) [reply .github/PULL_REQUEST_TEMPLATE.md:6]: Aha, sorry, I was confused. 🤕 
Personally I like the extra space/air in between the sections but if you guys don't want it I can of course remove it.

**Comment by virrios** [comment]
I really like having a template! Nice 🎉

**Review by palmenhq — DISMISSED** [review]
Me gusta

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,17 +1,19 @@
-<!-- Please add the Jira issue id to the Pull Request title, like this: "[HVG-123] Make the Pull Request template great again" -->
+<!-- 
+If there is one, add the ID of the Jira issue where this issue is described to the Pull Request title.
```
Comment (virrios) [inline .github/PULL_REQUEST_TEMPLATE.md:2]: I think "where this issue is described" can be remove. I think it just makes the sentence a bit more difficult to read.
a suggestion would be "If a Jira issue exists, add the issue ID to the Pull Request title"

Comment (styrelius) [reply .github/PULL_REQUEST_TEMPLATE.md:2]: Sure it might be a bit lengthy 👍

**Review by gustaveen — APPROVED** [review]
🎉🎉🎉

**Review by virrios — APPROVED** [review]

**Pull request merged**